src/docs/admin/references/NodeSourceInfrastructureReference.adoc

```diff
@@ -652,6 +652,16 @@ The following fields are available to configure your Azure infrastructure.
 - *domain:* Corresponds to the Azure _domain_, _tenant ID_, or _directory ID_ of your Active Directory.
 - *subscriptionId:* The _ID_ of your Azure subscription.
 
+The next parameters of the Azure Billing Configuration section, allow you to configure the automatic cloud cost estimator, by considering all the Azure resources related to your reservation (virtual machines, disks,..). This mechanism relies on the Azure Resource RateCard and Usage APIs (https://docs.microsoft.com/en-us/azure/cost-management-billing/manage/usage-rate-card-overview).
```

Comment:
estimator, by => estimator. It is done by  (I think it is better to make the sentences shorter)
relies on the => relies on both, the

Reply:
Ok

Reply:
Not agree for your second comment.